File: FitToFit/FitToFit.Services/Database/Sale.cs
using System;
using System.Collections.Generic;

namespace FitToFit.Services.Database;

public partial class Sale
{
    public int SalaId { get; set; }

    public string Naziv { get; set; } = null!;

    public string? Velicina { get; set; }

    public byte[]? Slika { get; set; }

    public virtual ICollection<Termini> Terminis { get; set; } = new List<Termini>();
}

File: FitToFit/UI/fittofit_mobile/lib/models/sale.dart
import 'package:fittofit_mobile/models/termini.dart';
import 'package:json_annotation/json_annotation.dart';

part 'sale.g.dart';

@JsonSerializable()
class Sale {
  int salaId;
  String naziv;
  String? velicina;
  String? slika;
  List<Termini>? terminis = [];

  Sale(
      {required this.salaId,
      required this.naziv,
      this.velicina,
      this.slika,
      this.terminis});

  factory Sale.fromJson(Map<String, dynamic> json) => _$SaleFromJson(json);

  Map<String, dynamic> toJson() => _$SaleToJson(this);
}
